dffml/df/memory.py

```diff
@@ -1408,6 +1349,65 @@ def __init__(
                 else:
                     task.exception()
 
+    async def operations_parameter_set_pairs(
```

Comment:
Did you mean to move this to `MemoryOperationImplementationNetwork` rather than `MemoryOrchestratorContext`? (#452)

Reply:
@pdxjohnny #452 says is should be in `MemoryOrchestratorContext`,did i miss some disscussion on that?

Reply:
Ah okay i was getting thrown off by how the diff is displayed on GitHub, my bad, at first glance it looked like this got moved under `MemoryOrchestratorContext`.

We should be able to remove the `ictx`, `octx`, `rctx` from the method's arguments and access them via `self.ictx`, etc.